src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -439,7 +438,7 @@ def _sync_sso_to_v1_credentials(aws_session_profile):
         aws_access_key_id=credentials['AccessKeyId'],
         aws_secret_access_key=credentials['SecretAccessKey'],
         aws_session_token=credentials['SessionToken'],
-        expiration = parser.isoparse(credentials['Expiration']),
+        expiration=parser.isoparse(credentials['Expiration']),
```

Comment:
I think this might have to be `parser.parse` to handle both formats

Reply:
Fixed in a previous commit.